Update README.md (#876)
mattermost/mattermost-handbook: operations/workplace/people/working-at-mattermost/paid-time-off/README.md

### When you take time off \(full days only\) please do the following:

> 1. Communicate your Paid Time Off (PTO) by booking time off in Bamboo > Time Off > Paid Time Off

Reviewer: ```suggestion
1. Communicate your Paid Time Off (PTO) by booking time off in **BambooHR > Time Off > Paid Time Off**.
```

> 2. As long as you have entered your time off in Bamboo and you have arranged for coverage while you are gone, you are able to take your vacation without it being approved in BambooHR. Consecutive days off in excess of 14 require manager approval in Bamboo prior to the vacation starting. 

Reviewer: ```suggestion
2. As long as you have entered your time off in Bamboo, and you have arranged for coverage while you are gone, you are able to take your vacation without it being approved in BambooHR. Consecutive days off in excess of 14 require manager approval in Bamboo prior to the vacation starting. 
```

3. PTO covers any number of things - including vacations, stay-cations, moving days, bereavement, etc. 

> 4. PTO is in addition to official public holidays in your region of residence. In some countries where there are statutory vacation requirements, public holidays are a supplementary entitlement. When entering a public holiday in BambooHR > PTO, please indicate in the notes section that it is a Public Holiday to distinguish it from vacation. 

Reviewer: ```suggestion
4. PTO is in addition to official public holidays in your region of residence. In some countries where there are statutory vacation requirements, public holidays are a supplementary entitlement. When entering a public holiday in **BambooHR > Time Off > Paid Time Off**, please indicate in the Notes section that it is a Public Holiday to distinguish it from vacation time. 
```

> If time off is being taken for sickness, including physical and mental needs, please enter these days in BambooHR > Sick Time, to distinguish from other types of PTO. Follow the appropriate process according to the statutory requirements in your region of residence. If taking extended time off for medical leave or for other types of [leaves](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/leaves-of-absence), reach out to HR: hr@mattermost.com .

Reviewer: ```suggestion
If time off is being taken for sickness, including physical and mental needs, please enter these days in **BambooHR > Sick Time**, to distinguish from other types of PTO. Follow the appropriate process according to the statutory requirements in your region of residence. If taking extended time off for medical leave or for other types of [leaves](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/leaves-of-absence), reach out to HR: hr@mattermost.com.
```

Reviewer: @lynn915 - I don't currently see an option for Sick Time in the BambooHR system.

4. [Autodirect Mattermost message replies](https://docs.mattermost.com/help/settings/account-settings.html#automatic-direct-message-replies) and set your [out-of-office email response](https://docs.mattermost.com/help/settings/account-settings.html#automatic-direct-message-replies). 
6. Enjoy your time off!

### Where to go to see who's out:

* [Stand-up channel](https://community.mattermost.com/private-core/channels/stand-up)
* [BambooHR Who's Out Calendar](https://mattermost.bamboohr.com/calendar). You can create an [iCalendar Feed](https://help.bamboohr.com/hc/en-us/articles/229310127-Create-an-iCalendar-Feed) to link the Who's Out Calendar to your GCalendar. 

### Common Time Off Questions

1. **Is there really no limit?** There is no limit to how much time off you can take when your work is meeting or exceeding expectations. We strongly recommend a minimum of 2 weeks per year.

> 2. **Is PTO monitored?** PTO is not monitored but we encourage managers to ensure their DRs are taking at least the recommended minimum of 2 weeks off per year (and/or the statutory minimum in their region of residence).  Time off is reported in regions where there is a statutory requirement to do so. 

Reviewer: ```suggestion
2. **Is PTO monitored?** PTO is not monitored, but we encourage managers to ensure their direct reports are taking at least the recommended minimum of 2 weeks off per year (and/or the statutory minimum in their region of residence). Time off is reported in regions where there is a statutory requirement to do so. 
```
